components/Mailer/interfaces.rst

```diff
@@ -0,0 +1,75 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_mailer_model_email-interface:
+
+EmailInterface
+~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single Email.
+
+.. note::
+    This interface extends :ref:`component_resource_model_timestampable-interface`. |br|
+    For more detailed information go to `Sylius API EmailInterface`_.
```

Comment:
A lot of models extend that interface, perhaps we should include this note everywhere, instead of copying it all over the place? Thoughts? This change can be done later, just keep it in mind. We can update it everywhere at once.